ANDROID Only (New Arch): CameraRollModule is not abstract and does not override abstract method removeListeners(double) in NativeCameraRollModuleSpec

# Version: 
```
    "react-native": "0.73.6",
    "@react-native-camera-roll/camera-roll": "^7.5.0",
```


# Error:

```
> Task :react-native-camera-roll_camera-roll:compileReleaseJavaWithJavac FAILED
/Volumes/macmini/xxx/node_modules/@react-native-camera-roll/camera-roll/android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java:67: error: CameraRollModule is not abstract and does not override abstract method removeListeners(double) in NativeCameraRollModuleSpec
public class CameraRollModule extends NativeCameraRollModuleSpec {
       ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Volumes/macmini/xxx/node_modules/@react-native-camera-roll/camera-roll/android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollPackage.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-camera-roll_camera-roll:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --info option to get more log output.
> Run with --scan to get full insights.

BUILD FAILED in 46s
24 actionable tasks: 17 executed, 7 up-to-date
error Command failed with exit code 1.
```

Help me!

--- a/android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java
+++ b/android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java
@@ -1088,4 +1088,7 @@ public class CameraRollModule extends NativeCameraRollModuleSpec {
   public void getPhotoThumbnail(String internalID, ReadableMap options, Promise promise) {
     promise.reject("CameraRoll:getPhotoThumbnail", "getPhotoThumbnail is not supported on Android");
   }
+
+  public void addListener(String eventName) {}
+  public void removeListeners(double count) {}
 }

